src/devices/RelayBoard/RelayType.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace RelayBoard
+{
+    /// <summary>
+    /// Describes what type of relay a device is.
+    /// </summary>
+    public enum RelayType
```

Comment:
Relay type in addition to normal open and normal closure, but also a large number of such as SPDT, DPDT and so on. How should this be handled?

Reply:
I think that the only importance here is the fact the relay is normally open or normally close. The technology behind has no importance for the code side. At least that's how I've always used relay.

Reply:
for the extra parameters you should create them only in the device which has such option but they should not be part of generic abstraction